feat: Utilise create-discord-bot (#10013)
discordjs/discord.js: apps/guide/src/content/02-getting-started/02-setting-up-an-application.mdx

![Bot application](/assets/bot-user.png)

In this panel, you can give your bot a snazzy avatar, set its username, and make it public or private. Your bot's token will be revealed when you press the "Reset Token" button and confirm. Once you've done this, copy it and paste it in the .env file after _`DISCORD_TOKEN=`_.

> If you happen to lose your bot's token at some point, you will need to come back to this page and reset your bot's token which will reveal the new token, invalidating all old ones.

Reviewer: ```suggestion
If you happen to lose this token at some point, you will need to come back to this page and reset it, which will reveal the new token, invalidating all old ones.
```
